<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仓库管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>781</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>781</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="title_LogIn">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>登录界面</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_LogIn">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>380</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_UserName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>账号:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_PassWord">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>密码:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_UserName">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_PassWord">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>290</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>761</width>
        <height>401</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>751</width>
          <height>381</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>751</width>
          <height>381</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QTableWidget" name="tableWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>761</width>
          <height>381</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>781</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_search_1">
         <property name="text">
          <string>查询一</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_search_2">
         <property name="text">
          <string>查询二</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_search_3">
         <property name="text">
          <string>查询三</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>仓库管理系统</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_LogIn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_LogIn_onClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_search_1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_search1_onClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_search_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_search2_onClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_search_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_search3_onClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_LogIn_onClicked()</slot>
  <slot>btn_search1_onClicked()</slot>
  <slot>btn_search2_onClicked()</slot>
  <slot>btn_search3_onClicked()</slot>
 </slots>
</ui>
